README.md

```diff
@@ -27,8 +27,22 @@ Depending on the workflow of each team some of them may be disabled to suit thei
 ### Client app
 
 To install the client app:
- - build it in Xcode and copy to your applications directory
- - run `defaults write com.babylonhealth.WallEView Host <application address>` 
+ - build it in Xcode and copy to your applications directory or download the app attached to the latest GitHub release
+ - run `defaults write com.babylonhealth.WallEView Host <application address>` to set the url to the app
  - launch the app
 
 Iconography © https://dribbble.com/shots/2772860-WALL-E-Movie-Icons
+
+### Debugging
+
+Using https://dashboard.ngrok.com/get-started you can run the app locally and still get all incoming events from GitHub webhooks.
+
+- setup ngrok and start it, it will print out the public address from which all requests will be redirected to your localhost, i.e. https://randomnumber.ngrok.io
+
+- add a webhook to the repository where you want to test the app (https://github.com/babylonhealth/walle-debug):
```

Comment:
```suggestion
- add a webhook to the repository where you want to test the app:
```

Reply:
it's ok to mention it as we can use it for internal use, there is nothing there
```suggestion
- add a webhook to the repository where you want to test the app (https://github.com/babylonhealth/walle-debug for use by Babylon team members):
```